pytorch_lightning/trainer/trainer.py

```diff
@@ -325,6 +327,7 @@ def __init__(
         Trainer._log_api_event("init")
         self.state = TrainerState()
         distributed_backend = distributed_backend or accelerator
+        gpu_ids, tpu_cores = self._parse_devices(gpus, auto_select_gpus, tpu_cores)
```

Comment:
tpu_cores gets parsed to tpu_cores, but gpus gets parsed to gpu_ids?

Reply:
That was the behavior in the Accelerator connector. It's a refactor PR, with no extra unintended changes.